### PR TITLE
IBX-368: Image normalization command

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-368
| Versions      | 2.5+

### CODE REVIEW

**Review by mateuszbieniek — APPROVED** [review]

File: docs/guide/images.md [hunk 1]
```diff
@@ -194,6 +194,15 @@ Additionally you can provide two parameters:
 
     This command publishes a new version of each Content item it modifies.
 
+## Normalizing images
+
+If you use image files with unprintable UTF-8 characters, you may come across a problem with images not displaying.
```
Comment (dabrt) [inline docs/guide/images.md:199]: Do these unprintable UTF-8 characters occur in file names or file paths?
Can we say:
"If you use image files with unprintable UTF-8 characters in file names, you might come across a problem with images not displaying."

**Review by dabrt — APPROVED** [review]

File: docs/guide/images.md (hunk 1 above)
Comment (DominikaK) [reply docs/guide/images.md:199]: File names (when uploading images from external source), paths are handled by the platform. I'd assumed it's self-explanatory that we mean filesnames here, isn't it?

Comment (dabrt) [reply docs/guide/images.md:199]: I know I am nitpicking and I know we do not support vector files, but there could be a CDRm SVG, or any other vector file with with unprontable chars _inside_. This is why I wanted to stress the fact that we're talking about file names.
"If you use images with unprintable UTF-8 characters in file names..."

Comment (DominikaK) [reply docs/guide/images.md:199]: Ah, you're right, I didn't think about SVG-like files.
```suggestion
If you use image files with unprintable UTF-8 characters in file names, you may come across a problem with images not displaying.
```

**Pull request merged**